Doc/library/operator.rst

```diff
@@ -112,6 +112,9 @@ The mathematical and bitwise operations are the most numerous:
 
    Return *a* converted to an integer.  Equivalent to ``a.__index__()``.
 
+   .. versionchanged:: 3.10
+      The type of the result is always :class:`int`.
```

Comment:
It may be worth addng a sentence to explain how this is different from the previous behaviour, since I think it'll come as a surprise to many that this wasn't already true before.

Reply:
Could you please propose a wording? I have problems with formulating this in English.

Reply:
So maybe something like:

> The result always has exact type :class:`int`. Previously, the result could have been an instance of a subclass of `int`.

Reply:
Thank you!